syntax = "proto3";

package widget;

import "base/template.proto";
import "base/widget_commons.proto";
import "base/actions.proto";

option go_package = "github.com/hotstar/hs-core-ui-models-go/widget";
option java_multiple_files = true;
option java_package = "com.hotstar.ui.model.widget";

message LoginSuccessWidget {
    base.Template template = 1 [deprecated = true];
    base.WidgetCommons widget_commons = 2;
    
    Data data = 11;
    message Data {
        string text = 1;
        string user_identity = 2;
        repeated base.Actions.Action on_complete_actions = 3;
        string mobile_number = 4;
        string email_address = 5;
    }
    
    reserved 3 to 10;
}
